### PR TITLE
New package: networkmanager-dmenu-2.1.0

### PR DESCRIPTION
**Pull request by ishaanbhimwal**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/networkmanager-dmenu/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'networkmanager-dmenu'
+pkgname=networkmanager-dmenu
+version=2.1.0
+revision=1
+depends="python3 NetworkManager python3-gobject"
+short_desc="Control NetworkManager via dmenu"
+maintainer="Ishaan Bhimwal <ishaanbhimwal@protonmail.com>"
+license="MIT"
+homepage="https://github.com/firecat53/networkmanager-dmenu"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/networkmanager-dmenu/template:10]: expand $homepage here

**Comment by classabbyamp** [comment]
the commit message should be just `New package: networkmanager-dmenu-2.1.0`

**Comment by ishaanbhimwal** [comment]
> the commit message should be just `New package: networkmanager-dmenu-2.1.0`

done!

**Comment by classabbyamp** [comment]
the rest of the message (from squashing) isn't necessary; it doesn't really add anything

**Comment by ishaanbhimwal** [comment]
done! (2)

**Comment by archbtw24** [comment]
check

File: srcpkgs/networkmanager-dmenu/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+license="MIT"
+homepage="https://github.com/firecat53/networkmanager-dmenu"
+distfiles="https://github.com/firecat53/networkmanager-dmenu/archive/refs/tags/v${version}.tar.gz"
+checksum=f39090bb924969dd5e76e8b07dd57ba89f146f444f225106383345d458de0729
+
+do_install() {
+	vbin networkmanager_dmenu
+	vlicense LICENSE.txt
+	vdoc README.md
+	vdoc config.ini.example
```
Comment (classabbyamp) [inline srcpkgs/networkmanager-dmenu/template:17]: this should probably be `vsconf` (see manual.md)

Comment (ishaanbhimwal) [reply srcpkgs/networkmanager-dmenu/template:17]: it makes sense to put those in `/usr/share/examples` but unfortunately the original package doesn't do that and doing so will ultimately confuse the user about where to find what.

* [example of a similar void linux template](https://github.com/void-linux/void-packages/blob/master/srcpkgs/clipmenu/template)
* [original package aur pkgbuild](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=networkmanager-dmenu-git)

Comment (ishaanbhimwal) [reply srcpkgs/networkmanager-dmenu/template:17]: it makes sense to put those in `/usr/share/examples` but unfortunately the original package doesn't do that and doing so will ultimately confuse the user about where to find what.

* similar void linux template [[1]](https://github.com/void-linux/void-packages/blob/master/srcpkgs/clipmenu/template)
* original package aur pkgbuild [[2]](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=networkmanager-dmenu-git)

Comment (classabbyamp) [reply srcpkgs/networkmanager-dmenu/template:17]: [1] doesn't install any example configuration at all
 
[2] is not void, arch may have different standards for package file locations (and that's AUR, which is more of a free-for-all than void-packages)

i still think that example configuration files should go in the place designated for example configuration files

Comment (ishaanbhimwal) [reply srcpkgs/networkmanager-dmenu/template:17]: your idea seems right considering the fact that arch doesn't even have a `/usr/share/examples` directory. can you confirm these changes so I can commit them:

```
--- a/srcpkgs/networkmanager-dmenu/template
+++ b/srcpkgs/networkmanager-dmenu/template
@@ -14,6 +14,6 @@ do_install() {
 	vbin networkmanager_dmenu
 	vlicense LICENSE.txt
 	vdoc README.md
-	vdoc config.ini.example
+	vconf config.ini.example
 	vinstall networkmanager_dmenu.desktop 644 /usr/share/applications
 }
```

Comment (ishaanbhimwal) [reply srcpkgs/networkmanager-dmenu/template:17]: your idea seems right considering the fact that arch doesn't even have a `/usr/share/examples` directory. can you confirm these changes so I can commit them:

```diff
--- a/srcpkgs/networkmanager-dmenu/template
+++ b/srcpkgs/networkmanager-dmenu/template
@@ -14,6 +14,6 @@ do_install() {
 	vbin networkmanager_dmenu
 	vlicense LICENSE.txt
 	vdoc README.md
-	vdoc config.ini.example
+	vconf config.ini.example
 	vinstall networkmanager_dmenu.desktop 644 /usr/share/applications
 }
```

Comment (classabbyamp) [reply srcpkgs/networkmanager-dmenu/template]: correct

**Comment by ishaanbhimwal** [comment]
@classabbyamp all these stuff should come under `do_install()` or `post_install()`?

```
vbin networkmanager_dmenu
vlicense LICENSE.txt
vdoc README.md
vsconf config.ini.example
vinstall networkmanager_dmenu.desktop 644 /usr/share/applications
```

**Pull request closed without merging**